core/terraform/variables.tf

```diff
@@ -250,6 +250,18 @@ variable "encryption_kv_name" {
   default     = ""
 }
 
+variable "enable_dns_policy" {
+  type        = bool
+  description = "Whether, or not, to add a DNS security policy with an allow-list. This is a preview feature that can be enabled to prevent data exfiltration via DNS."
+  default     = false
```

Comment:
Wouldn't be considered best to turn it on unless special circumstances? 
Maybe we don't eve want to offer the option to turn it off...
@askew @marrobi

Reply:
It's in public preview so not everyone will want it. That's the main driver. 

Also some of the workspace services need testing in more depth, although guess could be on in core in that instance.